easybuild/easyconfigs/d/Dalton/Dalton-2016_cmake-obey-EB-variables.patch

```diff
@@ -0,0 +1,76 @@
+--- dalton-release-2016-130ffaa0613bb3af6cac766fc8183d6df7d68917.orig/cmake/compilers/CFlags.cmake	2017-08-14 15:00:21.000000000 +0200
++++ dalton-release-2016-130ffaa0613bb3af6cac766fc8183d6df7d68917/cmake/compilers/CFlags.cmake	2018-02-06 15:12:30.449237790 +0100
+@@ -37,13 +37,13 @@
+ endif()
+ 
+ if(CMAKE_C_COMPILER_ID MATCHES Intel)
+-    set(CMAKE_C_FLAGS         "-g -wd981 -wd279 -wd383 -vec-report0 -wd1572 -wd1777 -restrict -DRESTRICT=restrict")
++	set(CMAKE_C_FLAGS         "${CMAKE_C_FLAGS} -wd981 -wd279 -wd383 -wd1572 -wd1777 -restrict -DRESTRICT=restrict")
+     if(NOT DEVELOPMENT_CODE)
+         # suppress warnings in exported code
+         set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -w")
+     endif()
+     set(CMAKE_C_FLAGS_DEBUG   "-O0")
+-    set(CMAKE_C_FLAGS_RELEASE "-O3 -ip")
++    set(CMAKE_C_FLAGS_RELEASE " ")
+     set(CMAKE_C_FLAGS_PROFILE "${CMAKE_C_FLAGS_RELEASE} -g -pg")
+     set(CMAKE_C_LINK_FLAGS "${CMAKE_C_LINK_FLAGS} -shared-intel")
+ 
+diff -ru dalton-release-2016-130ffaa0613bb3af6cac766fc8183d6df7d68917.orig/cmake/compilers/CXXFlags.cmake dalton-release-2016-130ffaa0613bb3af6cac766fc8183d6df7d68917/cmake/compilers/CXXFlags.cmake
+--- dalton-release-2016-130ffaa0613bb3af6cac766fc8183d6df7d68917.orig/cmake/compilers/CXXFlags.cmake	2017-08-14 15:00:21.000000000 +0200
++++ dalton-release-2016-130ffaa0613bb3af6cac766fc8183d6df7d68917/cmake/compilers/CXXFlags.cmake	2018-02-06 15:12:44.153908053 +0100
+@@ -39,7 +39,7 @@
+ endif()
+ 
+ if (CMAKE_CXX_COMPILER_ID MATCHES Intel)
+-    set(CMAKE_CXX_FLAGS         "-g -wd981 -wd279 -wd383 -vec-report0 -wd1572 -wd177 -fno-rtti -fno-exceptions")
++	set(CMAKE_CXX_FLAGS         "${CMAKE_CXX_FLAGS} -wd981 -wd279 -wd383 -vec-report0 -wd1572 -wd177 -fno-rtti -fno-exceptions")
+     if(DEVELOPMENT_CODE)
+         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
+     else()
+@@ -47,7 +47,7 @@
+         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -w")
+     endif()
+     set(CMAKE_CXX_FLAGS_DEBUG   "-O0")
+-    set(CMAKE_CXX_FLAGS_RELEASE "-O3 -ip")
++    set(CMAKE_CXX_FLAGS_RELEASE " ")
+     set(CMAKE_CXX_FLAGS_PROFILE "${CMAKE_CXX_FLAGS_RELEASE} -g -pg")
+     set (CMAKE_CXX_LINK_FLAGS "${CMAKE_CXX_LINK_FLAGS} -shared-intel")
+ 
+diff -ru dalton-release-2016-130ffaa0613bb3af6cac766fc8183d6df7d68917.orig/cmake/compilers/FortranFlags.cmake dalton-release-2016-130ffaa0613bb3af6cac766fc8183d6df7d68917/cmake/compilers/FortranFlags.cmake
+--- dalton-release-2016-130ffaa0613bb3af6cac766fc8183d6df7d68917.orig/cmake/compilers/FortranFlags.cmake	2017-08-14 15:00:21.000000000 +0200
++++ dalton-release-2016-130ffaa0613bb3af6cac766fc8183d6df7d68917/cmake/compilers/FortranFlags.cmake	2018-02-06 14:50:21.404219685 +0100
+@@ -48,13 +48,13 @@
+ 
+ if(CMAKE_Fortran_COMPILER_ID MATCHES Intel)
+     add_definitions(-DVAR_IFORT)
+-    set(CMAKE_Fortran_FLAGS         "-fpp -assume byterecl -DVAR_IFORT")
++    set(CMAKE_Fortran_FLAGS         "${CMAKE_Fortran_FLAGS} -fpp -DVAR_IFORT")
```

Comment:
Missed this one last time around.
I'm fairly certain you have to keep -assume byterecl for Intel fortran

Reply:
@akesandgren Not necessarily, the `DIRECT_IO_FACTOR` can be 1 or 4 with or without byterecl. There is a setting for `Dalton` I forget where it is, i vaguely remember that. Anyway, if you use the wring model, `Dalton will not work at all.